pkg/cmd/experimental/diagnostics/client.go

```diff
@@ -0,0 +1,45 @@
+package diagnostics
+
+import (
+	"fmt"
+
+	clientcmdapi "k8s.io/kubernetes/pkg/client/clientcmd/api"
+	"k8s.io/kubernetes/pkg/util"
+
+	clientdiagnostics "github.com/openshift/origin/pkg/diagnostics/client"
+	"github.com/openshift/origin/pkg/diagnostics/types"
+)
+
+const (
+	ConfigContexts = "ConfigContexts"
+)
+
+var (
+	AvailableClientDiagnostics = util.NewStringSet(ConfigContexts) // add more diagnostics as they are defined
```

Comment:
Since it's really only used in the package, I suppose it doesn't need to be public. If it's needed for integration tests or something later, we can make it that way.